src/disemvowel.js

```diff
@@ -0,0 +1,4 @@
+export const disemvowel = (string) => {
+  const stringFilter = string.toLowerCase().replace(/([aeiou])|(\d)|(\W)/g, '')
```

Comment:
this would be cleaner:
```js
export const disemvowel = (string) => {
  return string.toLowerCase().replace(/[aeiou\d\W]/g, '')
}
```

also I do not think you need to remove `\d` or `\W`. Consider reading the instructions again